app/controllers/mentors_controller.rb

```diff
@@ -146,16 +146,16 @@ def photo
   
   def auto_complete_for_mentor_fullname
     @mentors = Mentor.find(:all,
-                          :conditions => ["LOWER(firstname) LIKE :fullname
-                                            OR LOWER(lastname) LIKE :fullname
-                                            OR LOWER(display_name) LIKE :fullname
-                                            OR LOWER(uw_net_id) LIKE :fullname",
+                          :conditions => ["firstname LIKE :fullname
+                                            OR lastname LIKE :fullname
+                                            OR display_name LIKE :fullname
+                                            OR uw_net_id LIKE :fullname",
                                           {:fullname => "%#{params[:term].downcase}%"}])
```

Comment:
@mattharris5 This and the participant autocomplete get used quite a bit over the course of the day, and I'm on the fence about whether the initial wildcard should be removed like was done for event checkin. I'm not sure of many people who ever search by the middle of someone's name, but it might be useful if you're trying to find someone whose name you don't quite know.